docs/basic-setup.md

```diff
@@ -0,0 +1,68 @@
+# OpenCost Install
```

Comment:
@jessegoodier what are thoughts on moving these and other files in this directory to https://www.opencost.io/docs/?

Reply:
For sure, we need this there. Ideally that would source from this repo though?
I wonder if that is feasible?

Reply:
It's actually sourced from this repo currently: https://github.com/kubecost/opencost-website/tree/main/docs

That seem ok?